layers/+lang/lean/packages.el

```diff
@@ -0,0 +1,34 @@
+(setq dotspacemacs-additional-packages
+      (append '(dash dash-functional f s
+                     fill-column-indicator
+                     mmm-mode)))
```

Comment:
Add these packages to `lean-packages`. Never touch `dotspacemacs-additional-packages` in a layer.